Assignment09Starter/part02/index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 2</title>
+</head>
+<body>
+    
+    <?php
+        //Task 3.1: Change these items to reflect
+        //four of your favorite things
+        $my_favorite_things = array (
+            "1.Traveling",
+            "2.Photograpy",
+            "3.UX/UI design",
+            "4.Ballet"
```

Comment:
ideally this would just be an array of the things - not numbered. We should use our loop to do the numbering so we get the formatting advantage of an ordered list and if they switch order or the length of the array changes - our code can handle it.